kube/services/wts/wts-deploy.yaml

```diff
@@ -97,7 +97,7 @@ spec:
             path: /_status
             port: 80
           initialDelaySeconds: 30
-          periodSeconds: 60
+          periodSeconds: 20
```

Comment:
I think `livenessProbe` is actuaaly good, but it is `readinessProbe` taht are pinging WTS every 10 seconds, and we should change that probe to 20 or 30 seconds

Reply:
Great, thank you! The readinessProbe has been added.

Should the livenessProbe get reverted to 60 seconds?

Reply:
yeah lets put liveness probe back to 60